apps/artemis-web/components/hijack-table/hijack-table.tsx

```diff
@@ -508,26 +509,46 @@ const HijackTableComponent = (props) => {
             <b id="selected_hijacks_num">{hijackState.length}</b>
           </label>
         </span>
-        <select
-          onChange={(e) => setSelectState(e.target.value)}
-          style={{
-            width: '200px',
-            display: 'inline-block',
-            marginRight: '5px',
-          }}
-          className="form-control form-control-sm-auto"
-          id="action_selection"
-        >
-          <option value="hijack_action_resolve">Mark as Resolved</option>
-          <option value="hijack_action_ignore">Mark as Ignored</option>
-          <option value="hijack_action_acknowledge">
-            Mark as Acknowledged
-          </option>
-          <option value="hijack_action_acknowledge_not">
-            Mark as Not Acknowledged
-          </option>
-          <option value="hijack_action_delete">Delete Hijack</option>
-        </select>
+        {user && user.role === 'admin' ?
+          (<select
+            onChange={(e) => setSelectState(e.target.value)}
+            style={{
+              width: '200px',
+              display: 'inline-block',
+              marginRight: '5px',
+            }}
+            className="form-control form-control-sm-auto"
+            id="action_selection"
+          >
+            <option value="hijack_action_resolve">Mark as Resolved</option>
+            <option value="hijack_action_ignore">Mark as Ignored</option>
+            <option value="hijack_action_acknowledge">
+              Mark as Acknowledged
+            </option>
+            <option value="hijack_action_acknowledge_not">
+              Mark as Not Acknowledged
+            </option>
+            <option value="hijack_action_delete">Delete Hijack</option>
+          </select>)
+          :
+          (<select
+            onChange={(e) => setSelectState(e.target.value)}
+            style={{
+              width: '200px',
+              display: 'inline-block',
+              marginRight: '5px',
+            }}
+            className="form-control form-control-sm-auto"
+            id="action_selection"
+          >
+            <option value="hijack_action_acknowledge">
+              Mark as Acknowledged
```

Comment:
the simple user should also be able to ack/not ack groups of hijacks. Currently (tested) he can non-ack, but not ack groups. Please check locally on test installation.

Reply:
I think I fixed it now :) check again!

Reply:
yes it works. Resolving.